File: WarehouseManagementWeb.Infrastructure/Identity/Models/TokenInput.cs
namespace WarehouseManagementWeb.Infrastructure.Identity.Models
{
    /// <summary>
    /// Класс входной модели токенов пользователя.
    /// </summary>
    public class TokenInput
    {
        /// <summary>
        /// Токен доступа пользователя.
        /// </summary>
        public string? AccessToken { get; set; }

        /// <summary>
        /// Токен обновления пользователя.
        /// </summary>
        public string? RefreshToken { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/auth/models/input/token-input.ts
/**
 * Класс входной модели токенов пользователя.
 */
export class TokenInput {
  /**
   * Токен доступа пользователя.
   */
  accessToken: string = '';

  /**
   * Токен обновления пользователя.
   */
  refreshToken: string = '';
}
